ddtrace/contrib/internal/asgi/middleware.py

```diff
@@ -33,6 +33,7 @@
         service_name=config._get_service(default="asgi"),
         request_span_name="asgi.request",
         distributed_tracing=True,
+        obfuscate_404_resource=os.getenv("DD_ASGI_OBFUSCATE_404_RESOURCE", default=False),
```

Comment:
Can you switch the check to use `_get_config`, ie: 
https://github.com/DataDog/dd-trace-py/blob/9cc7814f4722b6f9be48a345d808ef8a13abfc88/ddtrace/contrib/internal/rq/patch.py#L16 ?

https://github.com/DataDog/dd-trace-py/blob/9cc7814f4722b6f9be48a345d808ef8a13abfc88/ddtrace/contrib/internal/rq/patch.py#L23